packages/framework-core/src/booster-token-verifier.ts

```diff
@@ -46,8 +43,17 @@ export class BoosterTokenVerifier {
         })
       }
 
+      let keys: jwt.Secret | jwt.GetPublicKeyOrSecret = getKey
```

Comment:
By default the token verifier will rely on jkws file located in a pubic url. If that url wasn't configured using `jkwsUri` we'll check the `publicKey` instead.